### PR TITLE
Feature/MZ-534

### PR DESCRIPTION
**Pull request by andrerussowsky**
# Midaz Pull Request Checklist

## Pull Request Type
[//]: # (Check the appropriate box for the type of pull request.)

- [ ] Ledger
- [ ] Auth
- [ ] Mdz
- [x] Transaction
- [ ] Pipeline
- [ ] Documentation

## Checklist
Please check each item after it's completed.

- [x] I have tested these changes locally.
- [ ] I have updated the documentation accordingly.
- [x] I have added necessary comments to the code, especially in complex areas.
- [x] I have ensured that my changes adhere to the project's coding standards.
- [x] I have checked for any potential security issues.
- [x] I have ensured that all tests pass.
- [ ] I have updated the version appropriately (if applicable).
- [x] I have confirmed this code is ready for review.

## Additional Notes
[//]: # (Add any additional notes, context, or explanation that could be helpful for reviewers.)

### CODE REVIEW

File: components/transaction/internal/ports/transaction.go [hunk 1]
```diff
@@ -152,6 +184,53 @@ func (handler *TransactionHandler) GetTransaction(c *fiber.Ctx) error {
 	return commonHTTP.OK(c, tran)
 }
 
+func (handler *TransactionHandler) GetAllTTransactions(c *fiber.Ctx) error {
```
Comment (MartinezAvellan) [inline components/transaction/internal/ports/transaction.go:187]: pls, adjust this method name. :)

File: components/transaction/internal/app/query/get-all-metadata-transactions.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+	"github.com/LerianStudio/midaz/common"
+	"github.com/LerianStudio/midaz/common/mlog"
+	commonHTTP "github.com/LerianStudio/midaz/common/net/http"
+	"github.com/LerianStudio/midaz/components/transaction/internal/app"
+	t "github.com/LerianStudio/midaz/components/transaction/internal/domain/transaction"
+	"github.com/google/uuid"
+)
+
+// GetAllMetadataTransactions fetch all Transacntions from the repository
```
Comment (MartinezAvellan) [inline components/transaction/internal/app/query/get-all-metadata-transactions.go:16]: pls, adjust this comment. :)

**Review by MartinezAvellan — APPROVED** [review]
lgtm

**Review by lfbarrile01 — APPROVED** [review]
approved

**Review by qnen — APPROVED** [review]
approved.

**Pull request merged**